### PR TITLE
Flexible release and bump

### PR DESCRIPTION
**Pull request by matzew**
For tests, I prefer building/pushing to my own "org"

Oh, and - should we now bump?  To avoid "clashes" ?

### CODE REVIEW

File: hack/release.sh [hunk 1]
```diff
@@ -4,11 +4,13 @@ set -ex
 
 pushd $(dirname "$0")/..
 
+readonly HUB_ORG="${HUB_ORG:-"openshift-knative"}"
```
Comment (jcrossley3) [inline hack/release.sh:7]: Nice!

File: version/version.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.5.0"
+	Version = "0.6.0"
```
Comment (jcrossley3) [inline version/version.go:4]: Normally, I'd do this in the same commit as adding the upstream file to `deploy/resources` but I don't guess it matters.

**Review by jcrossley3 — APPROVED** [review]
/lgtm

File: version/version.go (hunk 2 above)
Comment (matzew) [reply version/version.go:4]: I am a fan of maven-release-plugin :)   which bumps to the next version (with `-SNAPSHOT`) _after_ the actual releae is out - so the "old" (previous) release version string is no longer messing around w/ the "master" code

**Pull request merged**